core/src/main/java/com/linecorp/armeria/server/RoutingContext.java

```diff
@@ -61,6 +63,11 @@
     @Nullable
     String query();
 
+    /**
+     * Returns the query retrieved from the request as an {@link HttpParameters}.
```

Comment:
- `the query` -> `the query parameters`
- `from the request` -> `from the request path` (because we do not look into the content)
- Could remove `as an ...`?